templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,63 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo ${ip_hint}
+    }
+
+    try_to_bind_address() {
+      local ip_hint_file="$1"
+      ip_hint=$(get_ip_from_ip_hint_file "${ip_hint_file}")
+      if [[ -z "${ip_hint}" ]]; then
+        return
+      fi
+      while true; do
+        echo "Trying to bind ${ip_hint}"
+        exit_code=$(timeout 1s nc -l ${ip_hint}; echo $?)
```

Comment:
I tried this out locally on a F36 and I get this weird error
```
❯ nc -l 192.168.122.85
nc: getaddrinfo: Servname not supported for ai_socktype
```

Reply:
on rchos and fedora 33 i don't see such error.
can you try:
nc -ln 192.168.122.85

Reply:
Huh. I can't reproduce that on any of my systems, although I don't have F36 specifically. Apparently that error typically has something to do with /etc/services, but even if I intentionally break the permissions on that file I can't make it happen. :-/

Reply:
It works when I specify a port explicitly.
I think RHCOS ships with the nmap version of ncat and that might behave differently. I don't know about F33 but I got standard netcat on F36.
Should we specify the port explicitly to ensure compatibility with RHEL nodes?
```
nc -v -l "${ip_hint}" 31337
```

Reply:
That should be fine. We intentionally avoided specifying a port in the runtimecfg version of this workaround so we didn't have to worry about port conflicts. It's possible we could handle that here by randomizing the listen port each time through the loop.